sibylapp2/view/utils/filtering.py

```diff
@@ -106,6 +112,21 @@ def format_rowid_select(row_id):
     st.sidebar.metric(context.get_term("Prediction"), pred_display)
 
 
+def view_model_select(default=0):
+    if "model_id" in st.session_state:
+        st.session_state["select_model_index"] = st.session_state["model_ids"].index(
+            st.session_state["model_id"]
+        )
+    else:
+        st.session_state["select_model_index"] = default
+    st.sidebar.selectbox(
```

Comment:
I think, like with row ids, it would be less confusing to only show this selectbox if there are at least two models to choose from. I think this could be applied from here pretty easily with `if len(st.session_state["model_ids"] > 1`